databricks/resource_databricks_azure_adls_gen2_mount_test.go

```diff
@@ -49,14 +59,71 @@ func TestAccAzureAdlsGen2Mount_cluster_deleted_correctly_mounts(t *testing.T) {
 	})
 }
 
-func TestAccAzureAdlsGen2Mount_capture_error(t *testing.T) {
-	terraformToApply := testAccAzureAdlsGen2MountCaptureError()
+func TestAzureAccAdlsGen2Mount_capture_error(t *testing.T) {
+	if _, ok := os.LookupEnv("CLOUD_ENV"); !ok {
+		t.Skip("Acceptance tests skipped unless env 'CLOUD_ENV' is set")
+	}
+
+	clientID := os.Getenv("ARM_CLIENT_ID")             // make dual-env-var
+	clientSecret := os.Getenv("ARM_CLIENT_SECRET")     // make dual-env-var
+	tenantID := os.Getenv("ARM_TENANT_ID")             // make dual-env-var
+	subscriptionID := os.Getenv("ARM_SUBSCRIPTION_ID") // make dual-env-var
+	workspaceName := os.Getenv("TEST_WORKSPACE_NAME")
+	resourceGroupName := os.Getenv("TEST_RESOURCE_GROUP")
+	managedResourceGroupName := os.Getenv("TEST_MANAGED_RESOURCE_GROUP")
+	location := os.Getenv("TEST_LOCATION")
+	gen2AdalName := os.Getenv("TEST_GEN2_ADAL_NAME")
```

Comment:
test should assert if the env variables are set since they are required for the test to work.

Reply:
this test was completely rewritten, so should be covered there